src/theory/arith/nl/ext/factoring_check.cpp

```diff
@@ -35,7 +35,6 @@ namespace nl {
 FactoringCheck::FactoringCheck(Env& env, ExtState* data)
     : EnvObj(env), d_data(data)
 {
-  d_zero = NodeManager::currentNM()->mkConst(CONST_RATIONAL, Rational(0));
   d_one = NodeManager::currentNM()->mkConst(CONST_RATIONAL, Rational(1));
```

Comment:
What about that one?

Reply:
This is harder to address, I'll deal with this in another PR.